src/segundoProblema.js

```diff
@@ -0,0 +1,42 @@
+//RETO: Convertir código a ES6
+//Versión trabajada apartir del primer problema
+
+var API = 'https://rickandmortyapi.com/api/character/';
```

Comment:
En el problema 2 debíamos utilizar ECMAScript 6 por lo tanto ya no utilizamos 'var'

```suggestion
const API = `https://rickandmortyapi.com/api/character/`;
```